src/tests/ftest/util/daos_io_conf.py

```diff
@@ -41,6 +42,20 @@ def __init__(self, path="", filename="testfile", env=None):
         self.record_size = FormattedParameter("-s {}")
         self.obj_class = FormattedParameter("-O {}")
         self.filename = BasicParameter(None, filename)
+        self.mpi_type = "openmpi"
```

Comment:
Should this be configurable from a yaml setting?

Reply:
I made it an option that can be read for the test yaml and passed into the class init as the class itself defines daos_io_{gen|run}_conf command line arguments read form the test yaml.